ISISComputingGroup/ibex_gui
# Ticket2111 stress rig epics

### Description of work

Modify the Stress rig OPI (minor modifications) to support the EPICS IOC written for ticket 2111

### To test

https://github.com/ISISComputingGroup/IBEX/issues/2111

Ensure you also have the following changes when testing this:
https://github.com/ISISComputingGroup/EPICS-DeviceEmulator/pull/14
https://github.com/ISISComputingGroup/EPICS-ioc/pull/142
https://github.com/ISISComputingGroup/EPICS-instron/pull/1
https://github.com/ISISComputingGroup/EPICS/pull/83
https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/4

### Acceptance criteria

- [x] The OPI works with the IOC in ticket 2011 (for the functionality that has been implemented).

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
